content/quiz_process.md

```diff
@@ -7,19 +7,63 @@
+- If question 16 is a toss-up question and is erred, 16A is awarded as a 20-point assigned bonus question to the team who has not erred.  This bonus question goes to the corresponding chair of the team that has not erred.
+- Error points begin at question 17.
+- The first error committed, beginning at question 17, results in a toss-up question, which is numbered 17A, for the two remaining teams.  If there is an error on 17A, the next question will be 17B and will be a 10-point bonus for the team that has not committed an error.  This process applies for the remainder of the quiz.  Three teams must be enlisted to jump on questions 17, 18, 19, and 20.
 
 ### Tie-Breaker Quiz
+- In the event three teams are involved, a 20-question tie-breaker quiz will be used.
+    - The above rules for a three-team, 20-question quiz applies.
+- In the event two teams are involved, a 15-question tie-breaker quiz will be used with the following guidelines:
+    - If an error occurs on questions 1 through 11, the next numbered question is a 20-point assigned bonus to the team who has not erred.  This bonus goes to the corresponding chair of the team that has not erred.   
+    - If questions 12, 13, 14, or 15 are erred, the bonus question will remain on the same question number and will not be counted in the basic 15 questions. Both teams must jump on questions 13, 14, and 15.
+    - All bonus questions up to and including 12B are worth 20 points.
+    - All bonus questions from 13 to the end of the quiz are worth 10 points.
+    - Error points begin at question 13.
+    - A quiz-out is 3 correct questions.
 
 ### 2-Team 20-Question Quiz
+- There will be 20 questions in a quiz.
+- All questions have a value of 20 points when two teams are eligible.
+- If a question is missed, a bonus question will be read for the corresponding chair on the other team.
+- The bonus question will be the next numbered question, except for questions 16 and following.
+- All bonus questions throughout the whole quiz are 10 points. 
+- Both teams must be eligible to jump on questions 17 to 20.
 
 ## Platform Protocol
 
     Set the readiness bonus to 20.
 
-## Jumping
+- Quizzers will be seated facing the audience and quizmaster.
+- Feet of the quizzers are to be on the floor while jumping. 
+- A line-up card showing the seating order of each quizzer should be given to the quizmaster or scorekeeper before the start of each quiz.
+- No verbal or non-verbal communication is permitted between quizzers, coaches, or the audience from the time the quizmaster calls introduces the question type until points have been awarded.
+- Only a coach from a team presently seated on the platform, or the meet director may confer with the officials.
+- Coaches may have quiz material(s) on the platform during a time-out or prior to a quiz.
+- Quizzers will not touch the officials table.
+- Push-button quizzing can be provided by a district, if needed by physically disabled quizzers.
 
-### Pre-Jumping
```

Comment:
I removed this language from the header, and there's really not content for it. I "renamed" pre-jump to just "valid jump".....there's no need to define a distinction between jumping before the full question is read and after.

Reply:
Agreed.